packages/units/src/getBoundRect.ts

```diff
@@ -67,13 +67,13 @@ function getBoundRect(list: ElementBox[]) {
 }
 
 export function calcRotatedObjectPositionAndSize(
-  angle: number,
+  angle = 0,
   x: number,
   y: number,
   width: number,
   height: number
 ) {
-  if (!angle || angle === 0) {
+  if (!angle) {
```

Comment:
Hm, now if `angle` is `0`, `!angle` is true. It should still have `angle === 0` check as well, no?

Reply:
`!angle` does cover the 0 case too though? Or am I missing something?

Reply:
Oh, right, I was thinking something completely off.